src/Models/IoTWorkbenchProjectBase.ts

```diff
@@ -260,7 +260,16 @@ export abstract class IoTWorkbenchProjectBase {
   abstract async create(
       rootFolderPath: string, templateFilesInfo: TemplateFileInfo[],
       projectType: ProjectTemplateType, boardId: string,
-      openInNewWindow: boolean): Promise<boolean>;
+      openInNewWindow: boolean): Promise<void>;
+
+  /**
+   * Configure project environment: Scaffold configuration files with the given
+   * template files and open project properly.
+   */
+  abstract async configureProjectEnv(
+      channel: vscode.OutputChannel, scaffoldType: ScaffoldType,
+      configureRootPath: string, templateFilesInfo: TemplateFileInfo[],
+      openInNewWindow: boolean, customizeEnvironment: boolean): Promise<void>;

```

Comment:
>customizeEnvironment: boolean [](start = 32, length = 29)

Seems the "customizeEnvironment" is not always used in the inherited class.

Reply:
Fixed.